docs/source/markdown/podman-remote.1.md

```diff
@@ -17,7 +17,7 @@ Podman uses Buildah(1) internally to create container images. Both tools share i
 (not container) storage, hence each can use or manipulate images (but not containers)
 created by the other.
 
-Podman-remote provides a local client interacting with a Podman backend node through a varlink ssh connection. In this context, a Podman node is a Linux system with Podman installed on it and the varlink service activated. Credentials for this session can be passed in using flags, environment variables, or in `podman-remote.conf`
+Podman-remote provides a local client interacting with a Podman backend node through a RESTful API tunneled through a ssh connection. In this context, a Podman node is a Linux system with Podman installed on it and the API service activated. Credentials for this session can be passed in using flags, environment variables, or in `podman-remote.conf`
```

Comment:
```suggestion
Podman-remote provides a local client interacting with a Podman backend node through a RESTful API tunneled through a SSL/TLS connection. In this context, a Podman node is a Linux system with Podman installed on it and the API service activated. Credentials for this session can be passed in using flags, environment variables, or in `podman-remote.conf`
```

Reply:
No, we want them tunning through ssh not directly via SSL/TLS

Reply:
Thought we were using https?

Reply:
Nope.  ssh is default.